enigmatry-entry-blueprint-app/.vscode/extensions.json

```diff
@@ -0,0 +1,15 @@
+{
+    "recommendations": [
+        "stylelint.vscode-stylelint",
+        "dbaeumer.vscode-eslint",
+        "angular.ng-template",
+        "streetsidesoftware.code-spell-checker",
+        "waderyan.gitblame",
+        "eamodio.gitlens",
+        "davidanson.vscode-markdownlint",
+        "humao.rest-client",
+        "wayou.vscode-todo-highlight",
+        "mbinic.tgit-cmds",
```

Comment:
This is only useful for people using tortoise. Which seems pretty niche. Also, I don't think we want to push people in that direction, so I suggest to remove this one.

Reply:
Sure, but should we do then the same in our wiki?

Reply:
Then I suggest to update the wiki as well.